packages/stylelint-config/rules/extras.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  rules: {
+    // Reason: Avoid compound selectors creation using nested ampersand SASS syntax.
+    // Main reasons why use this pattern:
+    // - Unification with design system components code style
+    // - Look up easily for the selector using fulltext search in IDEs
+    // - Better orientation in longer component's code when scrolled down
+    // - Makes `no-descending-specificity` stylelint rule work correctly
+    // - IDE's support to click through from selector usage to its source
+    // Docs: https://stylelint.io/user-guide/rules/list/selector-nested-pattern/
+    'selector-nested-pattern': [
+      '(^&:)|(^&\\.)|(^&\\[)',
+      {
+        message: 'Only pseudo class selector, attribute selector and combination with state class is allowed',
+      },
+    ],
+
+    // Reason: Except for utility classes and third-party overrides, !important can be avoided.
+    // Docs: https://stylelint.io/user-guide/rules/list/declaration-no-important
+    'declaration-no-important': true,
+
```

Comment:
How about adding `declaration-property-value-disallowed-list`? We can avoid performance-heavy transitions that way.

https://github.com/visionappscz/stylelint-config/blob/master/index.js#L13

Reply:
You're right, there should be some list of properties that developers should think more before they code.